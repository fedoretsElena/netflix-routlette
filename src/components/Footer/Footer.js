import React from 'react';

import Logo from "../Logo/Logo";

const Footer = () => <footer className="p-3 bg-secondary text-center">
  <Logo />
</footer>

export default Footer;