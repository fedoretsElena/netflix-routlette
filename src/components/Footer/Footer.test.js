import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router';

import Footer from './Footer';

describe('Footer component', () => {
  it('should render Footer snapshot', () => {
    const { asFragment } = render(<MemoryRouter><Footer/></MemoryRouter>);
    expect(asFragment(<MemoryRouter><Footer/></MemoryRouter>)).toMatchSnapshot();
  })
})