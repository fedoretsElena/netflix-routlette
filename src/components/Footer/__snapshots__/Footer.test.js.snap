// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer snapshot 1`] = `
<DocumentFragment>
  <footer
    class="p-3 bg-secondary text-center"
  >
    <div
      class="logo f-size-18"
    >
       
      <a
        href="/"
      >
        <b>
          netflix
        </b>
        roulette
      </a>
    </div>
  </footer>
</DocumentFragment>
`;
